tests/testthat/apps/app/tests/shinytest/mytest-expected/001.json

```diff
@@ -0,0 +1,13 @@
+{
+  "input": {
+
+  },
+  "output": {
+    "map": [
```

Comment:
Woah, this is great: here you can specify the Shiny fields! Thanks for adding TDD in our Shiny app as well!